### DISCUSSION TITLE
Ошибка: доступ к видео ограничен

### DISCUSSION THREAD

**Discussion by Muhammadrizo04** [ilyhalight/voice-over-translation#955] [Общее]
![{05355CE5-655F-42BF-B039-926510177310}](https://github.com/user-attachments/assets/dee2cbf3-d906-4116-b50f-6cddbcdabc76)

здравствуйте, когда я пытаюсь посмотреть видео на русском, выдает такую ​​ошибку, почему если я захожу на другие видео, то оно их переводит

Hello, when I try to watch the video in Russian, it gives me this error. Why does it translate other videos when I go to them?

video: https://youtu.be/_SdYfz4hQtE?si=UsIcfi6EMq2R2uzF

**Comment by ilyhalight** [comment]
![firefox_q3Pv82K2Ud](https://github.com/user-attachments/assets/8434795a-fbdd-429f-afad-53c1b43fbdfc)

**Reply by Muhammadrizo04** [reply]
спасибо, все работает
